Rakefile

```diff
@@ -1,5 +1,5 @@
 require "bundler/gem_tasks"
-require 'metric_fu'
+require 'metric_fu' if RUBY_VERSION >= '2.1.0'
```

Comment:
No worries, I was more excited to learn some black magic ruby spell that I
don't actually know yet!

I'm not a huge fan of actually using `require 'bundler/setup'` and all of that,
especially inside gems, I prefer to require inside library code not indirectly
through the Gemfile via bundler. I hope that makes sense.

~Spaceghost